File: notification-service/src/main/java/com/eirsteir/coffeewithme/notification/NotificationServiceApplication.java
package com.eirsteir.coffeewithme.notification;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@EnableEurekaClient
@EnableJpaRepositories(basePackages = "com.eirsteir.coffeewithme.notification.repository")
@ComponentScan(
    basePackages = {
      "com.eirsteir.coffeewithme.commons.exception",
      "com.eirsteir.coffeewithme.notification"
    })
@SpringBootApplication
public class NotificationServiceApplication {

  public static void main(String[] args) {
    SpringApplication.run(NotificationServiceApplication.class, args);
  }
}

File: notification-service/src/main/kotlin/com/eirsteir/coffeewithme/notification/NotificationServiceApplication.kt
package com.eirsteir.coffeewithme.notification

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@EnableEurekaClient
@SpringBootApplication
class NotificationServiceApplication


fun main(args: Array<String>) {
    runApplication<NotificationServiceApplication>(*args)
}
